src/torchmetrics/retrieval/r_precision.py

```diff
@@ -22,17 +22,23 @@ class RetrievalRPrecision(RetrievalMetric):
 
     Works with binary target data. Accepts float predictions from a model output.
 
-    Forward accepts:
+    As input to ``forward`` and ``update`` the metric accepts the following input:
 
-    - ``preds`` (float tensor): ``(N, ...)``
-    - ``target`` (long or bool tensor): ``(N, ...)``
-    - ``indexes`` (long tensor): ``(N, ...)``
+    - ``preds`` (:class:`~torch.Tensor`): A float tensor of shape ``(N, ...)``
+    - ``target`` (:class:`~torch.Tensor`): A long or bool tensor of shape ``(N, ...)``
+    - ``indexes`` (:class:`~torch.Tensor`): A long tensor of shape ``(N, ...)`` which indicate to which query a
+      prediction belongs
 
-    ``indexes``, ``preds`` and ``target`` must have the same dimension.
-    ``indexes`` indicate to which query a prediction belongs.
-    Predictions will be first grouped by ``indexes`` and then `R-Precision` will be computed as the mean
+    .. note:: ``indexes``, ``preds`` and ``target`` must have the same dimension.
+
+    .. note:: Predictions will be first grouped by ``indexes`` and then `R-Precision` will be computed as the mean
```

Comment:
wondering if these `R-Precision` are meant as links or just nice formating...
if formatting it shall be double `` and if link I think they are not correct